### DISCUSSION TITLE
Trading Pokemon don't work

### DISCUSSION THREAD

**Discussion by shays7** [OnionUI/Onion#1635] [Support]
I have two mm+ devices and wanted to trade Pokemons, but when I tried it several times with the Crystal version on the GBC, it didn't work. Each time they synchronize, then I'm told that there are multiple save states but that the gambatte core uses them as the default. After confirming this information on both devices, it loads briefly and then tells me on the host device that no save states were received. Normal netplay on the NES works very well, though. Can anyone help me?? (version V4.3.1-1 of onion os used) 
Maybe it is because, I played only as long as I have two Pokemon on both devices each so I can trade??

**Comment by TimOelrichs** [comment]
The way Retroarch Netplay works is by sharing the savestates to sync the State of one emulated console. These means that both Players emulating the same Console, like when two Players sitting in front of a TV and playing NES. It was not designed with linkcable transfer in mind.  Gambatte has it's own netplayfeature in the core options. Never tried it, but maybe this will work for you. BTW i developed the core DoubleCherryGB which does linkcable transfer with Retroarch Netplay. It works with the Pokemon Games. It's currently not in OnionOS, but in Koriki CFW. Would be cool if the Onion Team will  add the core someday.
